pkg/k8sutil/portforward.go

```diff
@@ -164,7 +164,7 @@ func PortForward(localPort int, remotePort int, namespace string, getPodName fun
 			// Locks until stopChan is closed or connection to pod is lost.
 			// The main function may timeout before this returns an error
 			forwardErr = forwarder.ForwardPorts()
-			if forwardErr != nil {
+			if !errors.Is(forwardErr, portforward.ErrLostConnectionToPod) {
```

Comment:
Since `k8s.io/client-go` was changed to `v0.27.4`, this function now returns a specific error instead of handling it internally and returning nil when connection to a pod is lost, or stopChan is closed.

Old (v1.5.2): https://github.com/kubernetes/client-go/blob/v1.5.2/1.5/tools/portforward/portforward.go#L175

New (v0.27.4): https://github.com/kubernetes/client-go/blob/v0.27.4/tools/portforward/portforward.go#L235

As for _why_ `v0.x.y` is newer than `v1.x.y`, the [readme](https://github.com/kubernetes/client-go/tree/master#client-go) states that `v0.x.y` is for k8s >= 1.17.0